qark/plugins/Dynamically_Loading_Code.py

```diff
@@ -5,25 +5,58 @@
 from modules import common
 from lib.pubsub import pub
 import lib.plyj.parser as plyj
+import lib.plyj.model as m
 
 
 class DynamicallyLoadingCodePlugin(IPlugin):
     DEX_CLASS_LOADER = r'DexClassLoader'
     CLASS_LOADER = r'loadClass'
     DYNAMIC_BROADCAST_RECEIVER = r'registerReceiver'
+    global list_BR
```

Comment:
It needs to be "global" in order to make it accessible in the "recursive_register_receiver_function"
or else it will throw an error -
ERROR - Unable to run register receiver function plugin global name 'list_BR' is not defined